packages/create-app/src/lib/versions.ts

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Do we need to update docs for contributing to the repo to add something to this file?

Reply:
Possibly, would love to find a different solution xP

CI should fail if someone forgets to add a dependency, since the version will be missing and the templating should fail. I'm figuring we could see if it's a problem keeping it up to date or not.